src/features/game/mode/move/GameModeMove.ts

```diff
@@ -78,17 +80,19 @@ class GameModeMove implements IGameUI {
       maxYSpace: MoveModeConstants.button.ySpace
     });
 
+    let id = 0;
     moveMenuContainer.add(
-      buttons.map((button, index) =>
-        this.createMoveButton(
-          button.text,
+      buttons.map((button, index) => {
+        id++;
```

Comment:
Is there a reason to use `id` instead of the `index` that already exists?

Reply:
Deleted `id` and use `index` instead.